### ISSUE TITLE
Add german translation

### ISSUE THREAD

**Issue by ungo3ubs** [VladThodo/behe-keyboard#67]
[strings.txt](https://github.com/VladThodo/behe-keyboard/files/3748273/strings.txt)

here is the file for german translation. please change the file extension to xml. then you can show file correct.

i have found more words that were not in the xml file. Please add + add German Translation, because I can't.

NEXT -> WEITER
SKIP -> ÜBERSPRINGEN
GOT IT -> VERSTANDEN
Test your keyboard here  -> Teste deine Tastatur hier
SETTINGS -> EINSTELLUNGEN
SELECT -> AUSWÄHLEN
ENABLE -> AKTIVIEREN
Arrow keys on standard layout -> Pfeiltasten im Standardlayout
Number keys on standard layout -> Zifferntasten im Standardlayout
Key Height -> Tastenhöhe
NO THANKS -> NEIN DANKE
FOLLOW -> FOLGEN

**Comment by ThodeC** [comment]
I will work on this later. :)

**Comment by UltraBlackLinux** [comment]
Yes please I need this lol - also the keyboard layout would be good
